firebase-sessions/src/main/kotlin/com/google/firebase/sessions/FirebaseSessions.kt

```diff
@@ -141,5 +78,26 @@ internal constructor(
       } else {
         throw IllegalArgumentException("Firebase Sessions only supports the Firebase default app.")
       }
+
+    /**
+     * Lifecycle callbacks that will inform the [SessionLifecycleClient] whenever an [Activity] in
+     * this application process goes foreground or background.
+     */
+    private val activityLifecycleCallbacks =
```

Comment:
This has no reference to any instance anymore. Could this be extracted into a top level object in its own file?